canonical/ubuntu.com
# [Sunbeam GA Launch] Update OpenStack tab on /about/release-cycle

## Done

- Updated copy as per doc
- Added new d3 chart for MicroStack releases 
- Drive by:
  - Modularized release charts

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12944.demos.haus/about/release-cycle#openstack-release-cycle
- See that requested copy changes were made, check against [doc](https://docs.google.com/document/d/1-l3B7AoyDoETaL80UhlfEWCVKmMWPdy3zoF5LpsGcYQ/edit#)
- Check microstack chart and table on smaller screens against [release chart sheet](https://docs.google.com/spreadsheets/d/1KIuOAa620yseE0uhIl67u2SlS0bWlPI_d98bvYM2Yl8/edit#gid=949826076)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3612
